sdk/src/lib.rs

```diff
@@ -483,6 +483,11 @@ impl PortalSdk {
         Ok(descriptor)
     }
 
+    pub async fn show_mnemonic(&self) -> Result<(), SdkError> {
```

Comment:
This is a bit trickier to do, but maybe could you quickly try adding this new api to the react-native library as well? I think you can take inspiration from #29 which adds a few new APIs, in general the hardest part is converting arguments/return types, but in this case you can probably copy a function like `resume()` which doesn't take any arguments and just rename things around